src/Test/test_constraint.jl

```diff
@@ -528,8 +528,24 @@ function test_constraint_ZeroOne_bounds(
     @requires MOI.supports_constraint(model, MOI.VariableIndex, MOI.ZeroOne)
     x = MOI.add_variable(model)
     MOI.add_constraint(model, x, MOI.ZeroOne())
-    MOI.add_constraint(model, x, MOI.GreaterThan(T(0)))
-    MOI.add_constraint(model, x, MOI.LessThan(T(1)))
+    try
+        MOI.add_constraint(model, x, MOI.GreaterThan(T(0)))
+    catch err
+        if err isa MOI.LowerBoundAlreadySet
+            return
```

Comment:
I don't think this is right? It will skip the rest of the test. 

Shouldn't it be:
```julia
if !(err isa MOI.LowerBoundAlreadySet)
    rethrow(err)
end
```